cmd/check.go

```diff
@@ -216,7 +216,7 @@ func check() {
 				if !ok {
 					log.Println("Didn't find 'kubeconfig' in 'data-gatherers.k8s/pods' configuration. Assuming it runs in-cluster.")
 				}
-				k8sClient, err := k8s.NewClient(expandHome(kubeconfigPath))
+				k8sClient, err := k8s.NewDynamicClient(expandHome(kubeconfigPath))
```

Comment:
This is a nice trick. I didn't know about the existence of that `DynamicClient`.